src/test/java/com/google/api/gax/grpc/UnaryCallableTest.java

```diff
@@ -669,7 +666,6 @@ public void bundlingException() throws Exception {
         BundlingSettings.newBuilder()
             .setDelayThreshold(Duration.standardSeconds(1))
             .setElementCountThreshold(2L)
-            .setBlockingCallCountThreshold(0L)
```

Comment:
You're right. Since it's now null or positive, we should set this to null instead.